docs/api/commands/exec.md

```diff
@@ -44,7 +45,9 @@ This will create a new Bash session in the container `ubuntu_bash`.
 
 ## Divergence
 
-There is no known divergence between the Triton SDC Docker and Docker Inc. implementations of this method. Please contact Joyent support or file a ticket if you discover any.
+There is no known divergence between the Triton SDC Docker and Docker Inc.
+implementations of this method. Please contact Joyent support or file a ticket
```

Comment:
It's actually just fine to carry on hard-wrapping _Markdown Source_, which is what this file is.  The semantic construct of a paragraph, in Markdown, is a contiguous set of lines separated by a blank line.  This is essentially a code style thing, and is generally a per-repository decision.